### PR TITLE
[Region Migration]: Fix IoTV2 addRemotePeer stuck.

### PR DESCRIPTION
**Pull request by Pengzna**
as title.

### CODE REVIEW

**Comment by Pengzna** [comment]
@OneSizeFitsQuorum PTAL~

File: iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/pipe/consensuspipe/ConsensusPipeManager.java [hunk 1]
```diff
@@ -60,22 +63,65 @@ public ConsensusPipeManager(PipeConsensusConfig.Pipe config, ReplicateMode repli
...
-                String.valueOf(consensusPipeName.getSenderDataNodeId()))
-            .put(
-                EXTRACTOR_CONSENSUS_RECEIVER_DATANODE_ID_KEY,
-                String.valueOf(consensusPipeName.getReceiverDataNodeId()))
-            .put(EXTRACTOR_REALTIME_MODE_KEY, replicateMode.getValue())
-            .build(),
+        consensusPipeName.toString(), params.getLeft(), params.getMiddle(), params.getRight());
+  }
+
+  public void createConsensusPipe(
```
Comment (OneSizeFitsQuorum) [inline iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/pipe/consensuspipe/ConsensusPipeManager.java:72]: Do we necessarily know that an rpc sent to another node is not a coordinator? In this case, there is no need to add any parameters, and the sender and coordinator can be directly processed on the receiving side according to the inequality

Comment (OneSizeFitsQuorum) [reply iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/pipe/consensuspipe/ConsensusPipeManager.java:72]: ignore me

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM~
Awesome work！

**Pull request merged**